packages/manager/apps/dedicated/client/app/account/otrs/otrs.controller.js

```diff
@@ -6,6 +6,8 @@ import isDate from 'lodash/isDate';
 import map from 'lodash/map';
 import set from 'lodash/set';
 
+import moment from 'moment';
```

Comment:
`moment` is a global according to eslint and is imported with a script loader

Reply:
Have added it in `eslint` config.